formal-spec/Leios/SimpleSpec.agda

```diff
@@ -115,45 +121,54 @@ data _⇀⟦_⟧_ : Maybe LeiosState → LeiosInput → LeiosState × LeiosOutpu
        ────────────────────────────────────
        just s ⇀⟦ FTCH-LDG ⟧ (s , FTCH-LDG l)
 
-  -- TODO: Base chain
+  -- Base chain
+
+  Base₁ : ∀ {bs bs' eb txs} → let open LeiosState s in
```

Comment:
I think steps 1 and 2 should be separate. Storing transactions happens if `SUBMIT` arrives, but submitting an EB/transactions to the base chain happens once per slot (which is actually quite confusing to me, I think it should only happen once per slice maybe?). So both of these should be able to happen without the other.